### ISSUE TITLE
Release 1.2.4

### ISSUE THREAD

**Issue by josiah-wolf-oberholtzer** [agronholm/sphinx-autodoc-typehints#28]
Would it be possible to publish a 1.2.4 release to PyPI including the changes from #25?

**Comment by agronholm** [comment]
Yes, once I've polished the code. It's in progress.

**Comment by josiah-wolf-oberholtzer** [comment]
Great, thanks.

**Comment by agronholm** [comment]
If you could just resolve the big issues with the tests you contributed, I'd happily make a new release (with my code polishings included)

**Comment by josiah-wolf-oberholtzer** [comment]
Ah, this is a funny one. There's a bug in Sphinx's official testing fixtures where the dummy projects are copied to a temporary directory and Sphinx is run against them, but those projects are not cleaned-out on subsequent test runs, resulting in stale data.

A bug report is here: https://github.com/sphinx-doc/sphinx/issues/4040
A technique to fix this is here: https://github.com/pauleveritt/kaybee_prefactor/blob/5ee8adf8a5340e5c149b4a9acb3274a621cdc4bd/tests/integration/conftest.py#L21

Long story short, my changes _are_ running, but subsequent edits have no effect because Sphinx does isn't overwriting the dummy project.

I'll have a PR up shortly.

**Comment by josiah-wolf-oberholtzer** [comment]
See #29

**Comment by agronholm** [comment]
Once https://travis-ci.org/agronholm/sphinx-autodoc-typehints/builds/329682294 finishes, v1.2.4 should be available on PyPI. It's just taking forever to start.

**Comment by agronholm** [comment]
I manually uploaded the release since Travis seems to be having general difficulties with their container based infrastructure.
